src/layout/sideDrawer.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {FC, Dispatch, SetStateAction} from 'react'
+import {useHistory} from 'react-router-dom'
+import {
+  SwipeableDrawer,
+  Button,
+  List,
+  ListItem,
+  makeStyles,
+} from '@material-ui/core'
+import {routes} from '../routes'
+
+interface Props {
+  sideDrawOpen: boolean
+  setSideDrawerOpen: Dispatch<SetStateAction<boolean>>
+}
+
+export const SideDrawer: FC<Props> = ({sideDrawOpen, setSideDrawerOpen}) => {
+  const history = useHistory()
+
+  const useStyles = makeStyles(theme => ({
+    navList: {
+      width: 250,
+    },
+    navLink: {
+      padding: theme.spacing(1),
+    },
+  }))
+  const classes = useStyles()
+
+  const toggleDrawer = (open: boolean) => (
+    event: React.KeyboardEvent | React.MouseEvent,
+  ) => {
+    if (
+      event &&
+      event.type === 'keydown' &&
+      ((event as React.KeyboardEvent).key === 'Tab' ||
+        (event as React.KeyboardEvent).key === 'Shift')
```

Comment:
I would deal with this event in a function that accept an event and return a boolean

Reply:
Thanks very much for this Marco. Revisiting my code I realised that I had just copy/pasted from the material ui docs without completely understanding the code. The code you identified is not necessary and was performing a specific function within the docs demo that is not necessary in our context. We don't need access to the event which has greatly simplified the code.